### PR TITLE
[chore] Mark dotnet-specific metrics and attributes stable in yaml to match markdown

### PR DESCRIPTION
**Pull request by lmolkova**
## Changes

Follow-up on #781 - .NET md documents are marked stable, but yaml metrics and attributes are not.
This marks dotnet-specific (with corresponding prefix) attributes and metrics as stable. 

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] ~~[schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.~~

### CODE REVIEW

**Comment by joaopgrassi** [comment]
ping @open-telemetry/semconv-dotnet-approver

**Comment by antonfirsov** [comment]
I assume `dotnet-dns-metrics.md` and `dotnet-http-metrics.md` are not considered to be stable? I defer to @JamesNK for the server-side bits.

**Review by JamesNK — APPROVED** [review]

**Pull request merged**
